<launch>
	
	<arg name="map_file" default="$(find stdr_resources)/maps/lab.yaml" />
	
	<node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
	
	<node pkg="tf" type="static_transform_publisher" name="world2map" args="0 0 0 0 0 0  world map 100" />
		
	<rosparam file="$(find localization_project)/cfg/params.yaml" command="load" />
	
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find localization_project)/cfg/particles.rviz"/>
	
	<!-- node name="$(anon nao)" pkg="nao_localization" type="nao_interface.py" output="screen" -->
	
	<node pkg="localization_project" type="localization" name="localization" output="screen" />
	
	<node pkg="rosservice" type="rosservice" name="particles" args="call --wait /particles_initialization"/>

	<rosparam file="$(find nao_handler)/config/values_and_topics.yaml" command="load" />  

</launch>
